travis/self_tests

```diff
@@ -1,77 +1,577 @@
+        expected_errors.pop('duplicate-id-csv')
+        expected_errors.pop('copy-wo-api-one')
+        expected_errors.pop('manifest-version-format')
+        expected_errors.pop('manifest-required-author')
+        self.pylint_run(
+            is_pr=False, version="master",
+            expected_errors=27, errors_dict=expected_errors)
+
+    def check_v7(self):
+        """PYLINT_EXPECTED_ERRORS="84"
+           VERSION="7.0"
+        """
+        print "VERSION=7.0"
+        expected_errors = dict(EXPECTED_ERRORS)
+        expected_errors.pop('openerp-exception-warning')
+        expected_errors.pop('missing-readme')
+        expected_errors.pop('class-camelcase')
+        expected_errors.pop('manifest-deprecated-key')
+        expected_errors.pop('copy-wo-api-one')
+        expected_errors['manifest-version-format'] = 5
+        self.pylint_run(
+            is_pr=True, version="7.0",
+            expected_errors=84, errors_dict=expected_errors)
+
+        expected_errors = dict(EXPECTED_ERRORS)
+        expected_errors.pop('openerp-exception-warning')
+        expected_errors.pop('redundant-modulename-xml')
+        expected_errors.pop('attribute-deprecated')
+        expected_errors.pop('api-one-multi-together')
+        expected_errors.pop('except-pass')
+        expected_errors.pop('translation-field')
+        expected_errors.pop('sql-injection')
+        expected_errors.pop('wrong-tabs-instead-of-spaces')
+        expected_errors.pop('xml-syntax-error')
+        expected_errors.pop('wrong-import-order')
+        expected_errors.pop('manifest-required-key')
+        expected_errors.pop('too-complex')
+        expected_errors.pop('missing-manifest-dependency')
+        expected_errors.pop('missing-newline-extrafiles')
+        expected_errors.pop('incoherent-interpreter-exec-perm')
+        expected_errors.pop('no-utf8-coding-comment')
+        expected_errors.pop('duplicate-xml-record-id')
+        expected_errors.pop('missing-readme')
+        expected_errors.pop('missing-import-error')
+        expected_errors.pop('rst-syntax-error')
+        expected_errors.pop('use-vim-comment')
+        expected_errors.pop('translation-required')
+        expected_errors.pop('api-one-deprecated')
+        expected_errors.pop('invalid-commit')
+        expected_errors.pop('docstring-first-line-empty')
+        expected_errors.pop('class-camelcase')
+        expected_errors.pop('duplicate-xml-fields')
+        expected_errors.pop('manifest-deprecated-key')
+        expected_errors.pop('file-not-used')
+        expected_errors.pop('method-required-super')
+        expected_errors.pop('duplicate-id-csv')
+        expected_errors.pop('copy-wo-api-one')
+        expected_errors.pop('manifest-version-format')
+        expected_errors.pop('manifest-required-author')
+        self.pylint_run(
+            is_pr=False, version="7.0",
+            expected_errors=27, errors_dict=expected_errors)
+
+    def check_v8(self):
```

Comment:
Because this functions `chec_*` are called from a `test_*` function,  if we change the name, this will be computed in all builds of the matrix.